src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,26 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> treeSet = new TreeSet<>();
+        try (Reader reader = new FileReader(fileName)) {
+            int character = reader.read();
+            while (character != -1) {
+                if (Character.isLetter(character)) {
+                    treeSet.add((char) Character.toLowerCase(character));
+                }
+                character = reader.read();
+            }
+        } catch (FileNotFoundException e) {
+            throw new FileNotFoundException("File not found! " + e);
+        }
```

Comment:
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block

Reply:
done